migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -626,11 +626,9 @@ VALUES
     ('RFE', 'Record of First Nations Engagement', true, null, 'system-mds', 'system-mds'),
     ('TAL', 'Tenure Authorization Letter', true, null, 'system-mds', 'system-mds'),
     ('REV', 'Review', true, null, 'system-mds','system-mds'),
-    ('PUB', 'Public Comment', true, null, 'system-mds','system-mds'),
     ('CAL', 'Acknowledgement Letter', true, 'GDO', 'system-mds', 'system-mds'),
     ('WDL', 'Withdrawl Letter', true, 'GDO', 'system-mds', 'system-mds'),
     ('RJL', 'Rejection Letter', true, 'GDO', 'system-mds', 'system-mds'),
-    ('NPE', 'Permit Enclosed Letter', true, null, 'system-mds', 'system-mds'),
     ('PMT','Working Permit', true, null, 'system-mds','system-mds'),
```

Comment:
Don't we use the permit enclosed letter?

Reply:
these are dupes there are 2 versions of each